requirements.txt

```diff
@@ -17,3 +17,4 @@ docx2txt>=0.6
 lxml
 biopython # Enables Pubmed widget.
 ufal.udpipe >=1.2.0.3
+Orange3-Network
```

Comment:
Ok if we add this requirement to the add-on then two of the above comments can be ignored. :P Do we really want to push the user to install the other addon just because of one widget? I would prefer one of the other options:
- the widget does not show when Orange3-network not installed (current situation) - but then users would write issues why this widget not shown
- the widget shows error and warns the user to install Orange3-network.

What do you think @ajdapretnar

Reply:
The widget does not show. At least that was the current practice so far.

Reply:
So then this must be removed from requirements